Suggest to use a proper tag for app images (#3833)
vaadin/docs: articles/control-center/application-deployment/index.adoc

[source,shell]
----
docker build -t my-app:dev .
----

The `my-app:dev` tag is important because it's used later when deploying the application in Control Center.

If you're working with a local Kubernetes cluster (e.g., Minikube or Docker Desktop), there is no need to push the image to a container registry. However, if you're deploying to a remote cluster, the image needs to be pushed to a registry like Docker Hub:

[source,shell]
----
docker push company/my-app:dev
----

Make sure you have a Docker account, and that the image is tagged with a proper namespace (e.g., `company/my-app:dev`).

[TIP]

> It's common to use the application version to tag a release build (e.g., `my-app:1.0`). It's also common tag as `latest`, the most recent release. Using this tag forces Kubernetes always to pull the image from the registry, which requires the image to be there.

Reviewer: The second sentence makes less sense now. Perhaps replace with:
```suggestion
It's common to use the application version to tag a release build (e.g., `my-app:1.0`). It's also common to use the tag `latest`, representing the most recent release. Using this tag forces Kubernetes always to pull the image from the registry, which requires the image to be there.
```

[source,shell]
----
docker build -t my-app:dev .
----

The `my-app:dev` tag is important because it's used later when deploying the application in Control Center.

If you're working with a local Kubernetes cluster (e.g., Minikube or Docker Desktop), there is no need to push the image to a container registry. However, if you're deploying to a remote cluster, the image needs to be pushed to a registry like Docker Hub:

[source,shell]
----
docker push company/my-app:dev
----

Make sure you have a Docker account, and that the image is tagged with a proper namespace (e.g., `company/my-app:dev`).

[TIP]
====
It's common to use the app version to tag a release build, for example `my-app:1.0`.

> Another common tag is `latest` to mark the most recent release, but using this tag will force Kubernetes to always pull the image from the registry: deploying an application with the `latest` tag requires the image to be available in the registry.

Reviewer: ```suggestion
Another common tag is `latest` to mark the most recent release, but using this tag forces Kubernetes to always pull the image from the registry and requires the image to be available there.
```
